Cannot find "namespace_alias.hpp" when including <oneapi/mkl.hpp>

# Summary
Cannot find "namespace_alias.hpp" when including <oneapi/mkl.hpp> 

# Version
20241227

# Log
```
/.../onemkl_interfaces/20241227/include/oneapi/mkl.hpp:24:9: warning: Header `oneapi/mkl.hpp` is deprecated, please use `oneapi/math.hpp` instead [-W#pragma-messages]
   24 | #pragma message("Header `oneapi/mkl.hpp` is deprecated, please use `oneapi/math.hpp` instead")
      |         ^
/.../onemkl_interfaces/20241227/include/oneapi/mkl.hpp:34:10: fatal error: 'namespace_alias.hpp' file not found
   34 | #include "namespace_alias.hpp"
      |          ^~~~~~~~~~~~~~~~~~~~~
1 warning and 1 error generated.
```

--- a/include/oneapi/mkl.hpp
+++ b/include/oneapi/mkl.hpp
@@ -31,6 +31,6 @@
 #include "oneapi/math/rng.hpp"
 #include "oneapi/math/sparse_blas.hpp"
 
-#include "namespace_alias.hpp"
+#include "mkl/namespace_alias.hpp"
 
 #endif // ONEMATH_MKL_HPP

